src/test/java/teammates/test/cases/ui/InstructorCourseStudentDetailsEditSaveActionTest.java

```diff
@@ -133,7 +133,11 @@ public void testExecuteAndPostProcess() throws Exception{
         
         ______TS("Error case, invalid email parameter");
         
-        String invalidStudentEmail = "thisisaveryverylonglonglongstudentemailaccountname@gmail.tmt";
+        String invalidStudentEmail = "thisisaveryverylonglonglongveryverylonglonglong" 
+                                   + "veryverylonglonglongveryverylonglonglongveryvery"
+                                   + "longlonglongveryverylonglonglonglonglonglongveryverylonglonglong"
+                                   + "longlonglongveryverylonglonglonglonglonglongveryverlonglonglong"
+                                   + "studentemailaccountname@gmail.tmt";
```

Comment:
I have a memory of a utility method that can generate a random string of a specific length. May be use that so that the length of the string is visible to the reader. Also, you can use it to test the boundary case (i.e. 255 chars).